src/pextlib1.0/filemap.c

```diff
@@ -45,7 +45,6 @@
 #include <tcl.h>
 
 #include "filemap.h"
-#include "strcasecmp.h"
```

Comment:
This file now needs to `#include <strings.h>`.

Reply:
bah. you're right. compile tested but the test system has terrible header pollution

Reply:
also adding string.h for memcmp